jconstraints-z3/src/main/java/gov/nasa/jpf/constraints/solvers/nativez3/NativeZ3SolverContext.java

```diff
@@ -64,6 +67,10 @@
   private final Pattern bytePattern =
       Pattern.compile("(?:[^\\\\]|^)(?<toBeReplaced>\\\\x(?<digits>[0-9a-f]{2}))");
 
+  private boolean unsatTracking = false;
+  private int tracker = 0;
+  private Map<Expr, Expression> tracking = new HashMap<>();
```

Comment:
Unnecessary raw types, finality
```suggestion
  private final Map<Expr<BoolSort>, Expression<Boolean>> tracking = new HashMap<>();
```